tem(add): add page to explain reputation score + new concepts (#2014)
scaleway/docs-content: managed-services/transactional-email/concepts.mdx

</Message>

</Concept>

<Concept>
## Reputation score

A domain's **reputation score** is similar to a grade that email providers give to your domain's online reputation. The reputation score helps them decide whether the emails sent from your domain are trustworthy and should be delivered to your recipient's mailbox, or if your emails should be blocked, classified as spam and rejected. Email providers calculate your domain reputation on a scale of 0 to 100. 

The **closer to 100** your reputation score is, the more receiving email servers will trust your emails. A **high** domain reputation score means **better chances of your emails reaching the intended recipients' mailboxes**, while a **low** score might lead to **emails being marked as spam or rejected**.

<Message type="note">
A domain's reputation score is determined by email providers and Internet service providers (ISPs), which means you have a different domain reputation for each email service provider.
</Message>

</Concept>

<Concept>
## Send API

A send API is the main API that is used to send transactional emails to recipients.
</Concept>

<Concept>
## SMTP relay

> SMTP relay is the process of transferring an email from one mail server to another.

Reviewer: Maybe tweak it a bit like this
```suggestion
An SMTP relay is a service or server that helps in the transmission of email messages from one email server to another. They act as a bridge between the sender's mail server and the recipient's mail server.
SMTP relays are designed to handle large volumes of emails, ensuring that emails reach the recipient's server more efficiently and reliably, without being flagged as spam.
```

When a soft bounce occurs, the email software will try to resend the email for 72 hours (maximum) until the message is finally delivered.

</Concept>

<Concept>
## Spam trap

Spam traps are email addresses that Internet Service Providers (ISPs) and blocklist operators use to detect and blacklist spammers and senders who fail to adhere to email best practices. These email addresses may look like legitimate email addresses, but they are not operated by real users. 
There are two types or spam traps:
- [pristine spam traps](#pristine-spam-trap) and
- [recycled spam traps](#recycled-spam-trap).

Any contact that has not engaged by **opening** or **clicking an email** from you in the past year might be a spam trap.

<Message type="tip">
  Make sure you regularly clean up you contact lists to avoid spam traps.
</Message>

</Concept>

<Concept>
## SPF record

An SPF record specifies the mail servers that are allowed to send email for your domain. It allows you to protect senders and recipients from spam and phishing attacks. SPF also prevents spammers from sending emails on behalf of your domain.
</Concept>

<Concept>
## Transactional Email

> Transactional Email is a platform that allows you to send transactional emails. Unlike marketing emails, (which are mass distributions of the same marketing message to multiple recipients), transactional emails are personalized emails sent to individuals in response to events they have triggered (e.g. password recovery, billing information, delivery updates, etc).

Reviewer: ```suggestion
Transactional Email is a platform that allows you to send transactional emails. Unlike marketing emails (which are mass distributions of the same marketing message to multiple recipients), transactional emails are personalized emails sent to individuals in response to events they have triggered (e.g. password recovery, billing information, delivery updates, etc).
```

---

<Concept opened>
## Deliverability

Deliverability is the ability for an email to arrive in a recipient’s email inbox.
</Concept>

<Concept opened>
## DKIM record

A Domain Keys Identified Mail (DKIM) record is an email security standard. It provides you with an encryption key and a digital signature that allows you to verify whether an email was altered.
</Concept>

<Concept>
## DNS propagation

DNS propagation is the time a DNS change takes to be updated across the Internet.
</Concept>

<Concept>
## Hard bounce

> Hard bounce occurs when an email is sent to an invalid email address or an address that does not exist: unlike [soft bounce](#soft-bounce), which is a temporary error, hard bounce is a permanent error.

Reviewer 2: ```suggestion
Hard bounce occurs when an email is sent to an invalid email address or an address that does not exist. Unlike [soft bounce](#soft-bounce), which is a temporary error, hard bounce is a permanent error.
```

In this case, most emailing solutions will blacklist these email addresses.
</Concept>

<Concept>
## IP warming

IP warming is the process of gradually increasing the volume of emails sent through a new IP or an API that has not been used in a while.

> The process of warming up an IP address consists in gradually sending emails from it, starting with small volumes and increasing the volume over a period of days weeks or months, according to a set schedule.

Reviewer 2: ```suggestion
The process of warming up an IP address consists of gradually sending emails from it, starting with small volumes and increasing the volume over a period of days weeks or months, according to a set schedule.
```

Internet service providers are wary of emails sent from new IP addresses with no reputation, so warming up an IP allows for better email deliverability and thus, better reputation.

</Concept>

<Concept>
## IP reputation

> IP reputation measures the trustworthiness of an IP address and the amount of unwanted requests it sends. An IP address that sends authentic, spam-free emails, is assigned a good IP reputation. An IP address that sends high amounts of spam or viruses is assigned a bad IP reputation. IP reputation allows you to reject requests that are coming from an IP address with a bad reputation.

Reviewer 2: ```suggestion
IP reputation measures the trustworthiness of an IP address and the amount of unwanted requests it sends. An IP address that sends authentic, spam-free emails, earns a good IP reputation. An IP address that sends high amounts of spam or viruses earns a bad IP reputation. IP reputation allows you to reject requests that are coming from an IP address with a bad reputation.
```

Reviewer 2: I made these suggestions because to my ear it sounds a bit weird to say that a reputation can be "assigned". Alternatively you could replace "earns" with "gets" ?

SMTP relays are designed to handle large volumes of emails, ensuring that emails reach the recipient's server more efficiently and reliably, without being flagged as spam.
</Concept>

<Concept>
## Soft bounce

Soft bounce occurs when your email has been sent to a valid email address, and has reached the recipient's mail server but your email could not be delivered for multiple reasons:

- The recipient's mailbox has exceeded its storage limit
- The recipient's mail server is down
- The message was too large for the recipient's mailbox

When a soft bounce occurs, the email software will try to resend the email for 72 hours (maximum) until the message is finally delivered.

</Concept>

<Concept>
## Spam trap

Spam traps are email addresses that Internet Service Providers (ISPs) and blocklist operators use to detect and blacklist spammers and senders who fail to adhere to email best practices. These email addresses may look like legitimate email addresses, but they are not operated by real users. 
There are two types or spam traps:
- [pristine spam traps](#pristine-spam-trap) and
- [recycled spam traps](#recycled-spam-trap).

> Any contact that has not engaged by **opening** or **clicking an email** from you in the past year might be a spam trap.

Reviewer 3: "clicking an email"?

> Any contact that has not engaged by **opening** or **clicking an email** from you in the past year might be a spam trap.

Author: ```suggestion
Any contact that has not engaged by **opening** or **clicking a link sent within an email** from you in the past year might be a spam trap.
```
